ui-components/src/global/mixins.scss

```diff
@@ -1,3 +1,5 @@
+@use "sass:math";
```

Comment:
Changes here needed from the ag-grid update (the old division way is deprecated now)